Author: Kevin Chevreuil - Kaisen <kaisen@kaisenlinux.org>

--- adduser-3.118.orig/adduser.conf
+++ adduser-3.118/adduser.conf
@@ -36,10 +36,10 @@ LAST_SYSTEM_GID=999
 # FIRST_[GU]ID to LAST_[GU]ID inclusive is the range of UIDs of dynamically
 # allocated user accounts/groups.
 FIRST_UID=1000
-LAST_UID=59999
+LAST_UID=1010
 
 FIRST_GID=1000
-LAST_GID=59999
+LAST_GID=1010
 
 # The USERGROUPS variable can be either "yes" or "no".  If "yes" each
 # created user will be given their own group to use as a default.  If
